common/pinot/pinotQueryValidator.go

```diff
@@ -91,24 +91,56 @@ func (qv *VisibilityQueryValidator) validateWhereExpr(expr sqlparser.Expr) (stri
 	if expr == nil {
 		return "", nil
 	}
-	buf := sqlparser.NewTrackedBuffer(nil)
-
 	switch expr := expr.(type) {
 	case *sqlparser.AndExpr, *sqlparser.OrExpr:
 		return qv.validateAndOrExpr(expr)
 	case *sqlparser.ComparisonExpr:
 		return qv.validateComparisonExpr(expr)
 	case *sqlparser.RangeCond:
-		expr.Format(buf)
-		return buf.String(), nil
-		//return qv.validateRangeExpr(expr)
+		return qv.validateRangeExpr(expr)
 	case *sqlparser.ParenExpr:
 		return qv.validateWhereExpr(expr.Expr)
 	default:
 		return "", errors.New("invalid where clause")
 	}
 }
 
+func (qv *VisibilityQueryValidator) validateRangeExpr(expr sqlparser.Expr) (string, error) {
```

Comment:
Does this range query validator only handle (BETWEEN * AND *) query?

Reply:
Yes. <, > things are also handled validateComparisonExpr()

Reply:
Could you add comment for those 2 methods? Usually range query includes >,<,>=,<= these conditions, not just the "between" query.